domains/ash.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "ashleft",
+           "email": "",
+           "discord": "166692371981926400",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Ia-4ql1bWxICmtv44eSbttklIjIihW5o5erE8lDijwpZR2WP1ByAIAqTqF7B_H3UOM8BKhbanxazO1OpVsh68jZc_c99o0euWVGi8ut0cEuGZMR7P_7Ygja0yFipRRd5gWFQGZ6wLSkEhQX21X928MAUTqbt0QjH3nN80QVBuu77CrvpkiHsA598BAoReP-3I6ULnpSsae_Noyqpopc6XQq9zm9tnEJYSzp3ExaI3wH0c_mEckHcz--Tpjh_wqQMXfsjzs5Atn_2VzKoQbsQay8CEyWZOt4GtRTsv5CZrT0PEwZLH6cNHmibkPTgSAk5f0cm4V_3Fzr8c2iWeGla9w.VrhqtVXK-OnQEh-9XzwyQg.C08ijEwDm7VJ6ha9-oUVrm-S9I3rH33RiLly3UINv_nhomOzDo0kWYse1-zbo90Po1PEYlQ2wEpvEoEtCMZa2OUSZ9_5OLltDHeCM9e-Gog.-F878zXaKpweXDKHTWeqrg"
+        },
+    
+        "record": {
+            "TXT": "dh=aa69500fbfe5f77d83c58ff063396b458c1041fa"
```

Comment:
This is just being used for Discord verification when the site isn't displaying anything.